### PR TITLE
Add .NET Core

### PR DESCRIPTION
**Pull request by thangchung**
I create the awesome-dotnet-core collection, that includes all the frameworks, libraries, and resources of .NET core 1.0 that just released by Microsoft in June in this year.  

Please help me to update it into the awesome collection. Thank you very much!

-

# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [ ] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a dot, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please don't remove the pull request template. You're supposed to fill it out and have it completed before submitting a pull request. I've added it back. Please fill it out.

**Comment by thangchung** [comment]
@sindresorhus Thanks for your reminding. I just finished the checklist above. Please help me to check it again. Thanks :+1:

**Comment by sindresorhus** [comment]
Heading should be `Awesome .NET Core Collection` => `Awesome .NET Core`

-

> has a Table of Contents section **named Contents** as the first section,

Not followed.

-

Too many `A` and `An`'s in the descriptions. For example `A fast, small, full-featured IoC Container for .NET.` would read better as `Fast, small, full-featured IoC Container for .NET.`

-

`Best Articles` => `Articles`. It's an awesome list, so everything here should be "best".

-

> A collection of awesome .NET Core frameworks, libraries, tools, resources and software.

Linkify `.NET Core` to its website. The tagline should also be a blockquote.

**Comment by thangchung** [comment]
@sindresorhus I have fixed all suggestions from you. Sorry, this is my first FR into the awesome so that I took a lot of mistakes. Please help me to check once more time. Thanks!

File: readme.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [ColdFusion](https://github.com/seancoyne/awesome-coldfusion)
 - [Fortran](https://github.com/rabbiabram/awesome-fortran)
 - [.NET](https://github.com/quozd/awesome-dotnet)
+	- [.NET Core](https://github.com/thangchung/awesome-dotnet-core)
```
Comment (davisonio) [inline readme.md]: I would set this to `- [Core](https://github.com/thangchung/awesome-dotnet-core)`. It's under .NET item so it's implied that it's about .NET.

**Comment by davisonio** [comment]
Also, only the heading at the top of the readme should be using `# Title` the subsequent sections should be like `## Title`, `### Title` etc.

**Comment by thangchung** [comment]
@davisonio Thanks for your comment. I have fixed it. Please help me on that :+1:

**Comment by sindresorhus** [comment]
- It could use a `Community` section like this: https://github.com/sindresorhus/awesome-electron#community
- The repo link (on the right of the repo description) is linking to itself. Just remove it.
- You have some articles, but I'd also like to see some books, videos, talks, etc.
- Use the SVG version of the license badge. See the one used here.
- `.Net` => `.NET`
- `Comparison between dotnet core and nodejs` => `Comparison between .NET Core and Node.js`

**Comment by thangchung** [comment]
@sindresorhus I fixed all above, but one item I do not quite understand it 
> * Use the SVG version of the license badge. See the one used here.

Is it the awesome badge on the top of page? I already use the SVG format?

**Comment by sindresorhus** [comment]
> Is it the awesome badge on the top of the page? I already used the SVG format?

No, this: https://github.com/thangchung/awesome-dotnet-core#license

**Comment by thangchung** [comment]
@sindresorhus Sorry, I missed that one, just updated. Thank very much!

**Comment by sindresorhus** [comment]
> It could use a Community section like this: https://github.com/sindresorhus/awesome-electron#community

By this I obviously mean it should be filled in. Not empty sections ;)

-

`BCrypt.net` => `BCrypt.NET`

**Comment by sindresorhus** [comment]
> By this I obviously mean it should be filled in. Not empty sections ;)

All of the empty sections.

**Comment by thangchung** [comment]
I have done. Please check it. Thanks!

**Comment by sindresorhus** [comment]
Awesome! Added :)

**Pull request merged**